pkg/apis/pingcap/v1alpha1/tidbclusterautoscaler_types.go

```diff
@@ -112,6 +112,26 @@ type BasicAutoScalerSpec struct {
 	// If not set, the default metric will be set to 80% average CPU utilization.
 	// +optional
 	Metrics []v2beta2.MetricSpec `json:"metrics,omitempty"`
+
+	// MetricsTimeWindowSeconds describe the time window seconds for the metrics
+	// to be queried in the Prometheus.
+	// If not set, the default value would be 180.
+	// +optional
+	MetricsTimeWindowSeconds *int32 `json:"metricsTimeWindowSeconds,omitempty"`
```

Comment:
I prefer using the time duration here, e.g. metav1.Duration.

> Syntactically, a range duration is appended in square brackets ([]) at the end of a vector selector to specify how far back in time values should be fetched for each resulting range vector element. ...

https://prometheus.io/docs/prometheus/latest/querying/basics/#range-vector-selectors

Reply:
updated.